### DISCUSSION TITLE
[Blog] Life made easy feat. VS Code

### DISCUSSION THREAD

**Discussion by ashwinexe** [EddieHubCommunity/support#1832] [Show and tell]
Hey! My new article is here: This time less text more visuals to help anyone understand. Check out how I use shortcuts in my code editor to save  time and increase workflow: 
https://ashwinexe.medium.com/life-made-easy-feat-vs-code-84a570da18f3

**Comment by verma-kunal** [comment]
Resources Used:
To record gifs : An open source app called **ScreenToGif**
To show commands: An app called **WS Demo** (used on trial mode)

**Comment by ashwinexe** [comment]
Thanks for the support, I have updated the blog with few more (amazing) shortcut keys. 
Let us know what shortcuts you use in the comments :)

**Comment by AnadeOre** [comment]
Alt+Click lets you add multiple cursors not necessarily aligned!

**Reply by ashwinexe** [reply]
Thanks for clarifying. `Alt+click` can be used to place multiple cursor at different positions while `ctrl+alt+arrow` puts multi cursor aligned below and above current cursor.
